index.js

```diff
@@ -3,16 +3,49 @@ const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
+
 /***** Deliverable 2 *****/
 
+let headerColor = document.getElementById('header')
+headerColor.style.color = 'blue';
+
+
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+let foundDiv = document.querySelector('.traveler')
+    foundDiv.innerHTML +=
+    `<div id="profile">
+      <img src="${traveler.photo}"
+        alt="${traveler.name}">
+      <h2>${traveler.name}</h2>
+      <em>${traveler.nickname}</em>
+    </div>`
```

Comment:
Here, you don't want to use `+=`, since there is already some content in the `.traveler` element. A better approach for this problem is to find each element in the HTML and update its properties, like this:

```js
const img = document.querySelector("#profile img");
img.src = traveler.photo;
img.alt = traveler.name;

const name = document.querySelector("#profile h2");
name.textContent = traveler.name;

const nickname = document.querySelector("#profile em");
nickname.textContent = traveler.nickname;
```